main.tf

```diff
@@ -1,21 +1,16 @@
 module "label" {
```

Comment:
I think you can drop this module and just use module.this, since you don't have multiple labels.

Reply:
Good suggestion!